diff --git a/app/controllers/users_controller.rb b/app/controllers/users_controller.rb
index 2d8f818f..fd068dff 100644
--- a/app/controllers/users_controller.rb
+++ b/app/controllers/users_controller.rb
@@ -96,6 +96,11 @@ class UsersController < ApplicationController
       return redirect_to settings_users_path, notice: I18n.t('unable_to_remove_user')
     end
 
+    # Prevent deletion of the last admin
+    if @user.admin? && current_account.users.active.admins.where.not(id: @user.id).none?
+      return redirect_to settings_users_path, alert: 'Cannot remove the last admin user'
+    end
+
     @user.update!(archived_at: Time.current)
 
     redirect_back fallback_location: settings_users_path, notice: I18n.t('user_has_been_removed')
@@ -104,7 +109,7 @@ class UsersController < ApplicationController
   private
 
   def role_valid?(role)
-    User::ROLES.include?(role)
+    User::SELECTABLE_ROLES.include?(role)
   end
 
   def build_user
